stats/record.go

```diff
@@ -22,10 +22,24 @@ import (
 	"go.opencensus.io/tag"
 )
 
+func init() {
+	internal.SubscriptionReporter = func(measure string) {
```

Comment:
may as well accept Measure and set `internal.SubscriptionReporter = measure.subscribed`

Reply:
I can't, it creates a circular dependency.